setup_no_spack.sh

```diff
@@ -0,0 +1,74 @@
+# !/usr/bin/env bash
+
+test_folder="tiny_CI_test"
+
+export LIBSONATA_ZERO_BASED_GIDS=1
+export OMP_NUM_THREADS=1
+export SONATAREPORT_DIR=$(pwd)/libsonatareport/build/install
+export NEURODAMUS_NEOCORTEX_ROOT=$(pwd)/neurodamus-models/build/install
+export HOC_LIBRARY_PATH=$NEURODAMUS_NEOCORTEX_ROOT/share/neurodamus_neocortex/hoc
+export CORENEURONLIB=$NEURODAMUS_NEOCORTEX_ROOT/lib/libcorenrnmech.dylib
+export NRNMECH_LIB_PATH=$NEURODAMUS_NEOCORTEX_ROOT/lib/libnrnmech.dylib
+
+deactivate
+
+if [ -d "venv" ]; then
+  echo "Found existing venv directory. Just load env"
+  source venv/bin/activate
+else
+  python -m venv venv
+  source venv/bin/activate
+  pip install --upgrade pip
+  pip install NEURON-nightly cython
+  pip cache purge
+  MPICC=$(brew --prefix openmpi)/bin/mpicc pip install --no-binary=mpi4py mpi4py
+  export HDF5_INCLUDEDIR=$(brew --prefix hdf5-mpi)/include
+  export HDF5_LIBDIR=$(brew --prefix hdf5-mpi)/lib
+  python -m pip install --upgrade pip setuptools
+  CC="mpicc" HDF5_MPI="ON" HDF5_INCLUDEDIR=$HDF5_INCLUDEDIR HDF5_LIBDIR=$HDF5_LIBDIR \
+  pip install --no-cache-dir --no-binary=h5py h5py --no-build-isolation
+  pip install neurodamus morphio
+fi
+
+if [ ! -d "libsonatareport" ]; then
+    git clone git@github.com:openbraininstitute/libsonatareport.git --recursive --depth=1
```

Comment:
Can we use https for downloading? Since I didn't have my machine configured for ssh with github :)

Reply:
done. How do you push changes? github stopped accepting name/password